src/js/tracks/text-track.js

```diff
@@ -339,14 +344,17 @@ class TextTrack extends Track {
   addCue(originalCue) {
     let cue = originalCue;
 
-    if (!(originalCue instanceof window.vttjs.VTTCue)) {
+    if (window.vttjs && !(originalCue instanceof window.vttjs.VTTCue)) {
       cue = new window.vttjs.VTTCue(originalCue.startTime, originalCue.endTime, originalCue.text);
 
       for (const prop in originalCue) {
         if (!(prop in cue)) {
```

Comment:
I think since we're only copying the id, having it be separate with a comment makes it much nicer and explicit. Though, if we end up needing to force copy other stuff, I'd agree combining into here would make more sense.